src/app/components/index_home.jsx

```diff
@@ -43,9 +43,18 @@ class Index_home extends Component{
     return (
       <div>
         {this.renderName(this.props.currentUser)}
-        <div>
-            <b>About</b>
-            <p>This is the about paragraph.</p>
+        <div className="welcomeHome">
+            <div className="transBox">
+              <p>Here is the list of classes you are taking:</p>
+
+                <p>Class A</p>
+                <p>Class B</p>
+                <p>Class C</p>
+                <p>Class D</p>
+                <p>Class E</p>
```

Comment:
She just hard coded those in, I'm guessing as an example of what the output can look like. Once we get the courses added to each user in the database, it will be dynamic.